### PR TITLE
Fix the id used in aria-controls used to make a relationship between the popup and its parent

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/integration/accessibility_spec.js [hunk 1]
```diff
@@ -151,6 +151,22 @@ describe("accessibility", () => {
...
+        pages.map(async ([browserName, page]) => {
+          await page.waitForSelector(".annotationLayer");
+          const stampId = "pdfjs_internal_id_20R";
+          await page.click(`#${stampId}`);
+
+          const controledId = await page.$eval(
+            "#pdfjs_internal_id_21R",
+            el => document.getElementById(el.getAttribute("aria-controls")).id
+          );
+          expect(controledId).withContext(`In ${browserName}`).toEqual(stampId);
```
Comment (Snuffleupagus) [inline test/integration/accessibility_spec.js:165]: Spelling-nit:
```suggestion
          const controlledId = await page.$eval(
            "#pdfjs_internal_id_21R",
            el => document.getElementById(el.getAttribute("aria-controls")).id
          );
          expect(controlledId).withContext(`In ${browserName}`).toEqual(stampId);
```

**Review by Snuffleupagus — APPROVED** [review]
r=me, with the comment addressed; thank you!

File: test/integration/accessibility_spec.js (hunk 1 above)
Comment (calixteman) [reply test/integration/accessibility_spec.js:165]: It was my first draft... and I thought that it doesn't make sense to have two 'l' here.

**Pull request merged**